lib/nexpose/common.rb

```diff
@@ -25,20 +25,27 @@ module Scope
   # non-members.
   class Email
     # Send as file attachment or zipped file to individuals who are not members
-    # of the report access list. One of: file|zip
+    # of the report access list.
+    # [String] Attachment format, 'file' | 'zip'.
     attr_accessor :send_as
     # Send to all the authorized users of sites, groups, and assets.
+    # [Fixnum] 1 | 0
     attr_accessor :to_all_authorized
     # Send to users on the report access list.
+    # [String] Attachment format 'file' | 'zip'
     attr_accessor :send_to_acl_as
-    # Format to send to users on the report access list. One of: file|zip|url
+    # Format to send to users on the report access list.
+    # [String] Attachment format 'file' | 'zip' | 'url'
     attr_accessor :send_to_owner_as
-
     # Sender that e-mail will be attributed to.
+    # [String] an email address
     attr_accessor :sender
-    # SMTP relay server.
+    # SMTP relay server. This is necessary to avoid security issue for
```

Comment:
I generically wrote security reasons to specify spam and data transmission. If you suggest I can be more specific.